lib/reports/resolvers.ts

```diff
@@ -367,13 +372,37 @@ export const generateAccountantReport: Resolver<
     { id: 'donationAmount', title: 'Donation' },
     { id: 'totalAmount', title: 'Total' },
   ];
-  const csvWriter = createObjectCsvWriter({
-    path: 'temp/file.csv',
+
+  // Generate CSV string from csv object.
+  const csvStringifier = createObjectCsvStringifier({
     header: csvHeaders,
   });
-  await csvWriter.writeRecords(csvAccountantReportRows);
+  const csvStringRecords = csvStringifier.stringifyRecords(csvAccountantReportRows);
+  const csvStringHeader = csvStringifier.getHeaderString();
+  const csvString = csvStringHeader + csvStringRecords;
+
+  // NOTETOSELF: Change name
+  // CSV naming format applications/permit-holders/accounting-report-{employeeID}-{timestamp}.csv
+  const employeeID = session.id;
+  const timestamp = formatDateTimeYYYYMMDDHHMMSS(new Date());
+  // NOTETOSELF: Change fileName and s3InvoiceKey
+  const fileName = `accounting-report-${employeeID}-${timestamp}.csv`;
+  const s3InvoiceKey = `rcd/applications/permit-holderss/${fileName}`;
```

Comment:
Reused application report name as a placeholder. Any suggestions on what to name `fileName` and `s3InvoiceKey`?

Reply:
Yes, please name `fileName` as `accounting-report-{employeeID}-{timestamp}.csv` and the key as `rcd/reports/${fileName}`

Reply:
Addressed!